subcommands/waves/status.go

```diff
@@ -134,6 +134,23 @@ func doShowWaveStatus(cmd *cobra.Command, args []string) {
 			}
 		}
 	}
+
+	if len(status.RolloutHistory) > 0 {
```

Comment:
I have no strong opinion, but it seems to me that this information would look more organically in a `fioctl waves show` command.

Reply:
I think its fine. One concern I'm now having - this array of "history" could grow quite large, right? I know pagination is a contentious topic, but I have to ask if we need to consider something like that here?

Reply:
This thing is for one wave.

I can in theory imagine a user doing a thousand "wave rollout" commands for one wave.
It just looks so tedious the user would be tired way too fast to finish.

So, I'd prefer to first see that coming in reality.